x-pack/plugin/src/test/resources/rest-api-spec/test/data_stream/10_data_stream_resolvability.yml

```diff
@@ -32,6 +32,11 @@
         index: logs-foobar
   - match: { error.reason: "No searchable snapshots indices found"}
 
+  - do:
+      ilm.explain_lifecycle:
+        index: logs-foobar
+  - is_false: indices.logs-foobar.managed
```

Comment:
I am not sure how this can work? I would have thought only concrete indices and not the data stream name could appear in the output?

Reply:
The current asserts tests that the data stream name does not appear in the response.
I will add the following assert`- is_true: indices.logs-foobar-000001.managed` when addressing the previous comment.